docs/spec/nmt.md

```diff
@@ -60,11 +60,11 @@ For the sake of brevity, we have only included the first 7 hex digits of SHA256
               /            \                    /               \
             NsH()          NsH()              NsH()             NsH()
             /                \                /                   \
-    00 00 5fa0c9c       00 00 52385a0    01 01 71ca46a       03 03 b4a2792    Leaf Nodes
+    00 00 5fa0c9c       00 00 52385a0    01 01 71ca46a       03 03 b4a2792    Leaf Hashes
         |                   |                 |                   |
       NsH()               NsH()              NsH()               NsH()
         |                   |                 |                   |
-00 6c6561665f30      00 6c6561665f31    01 6c6561665f32      03 6c6561665f33  Namespaced Data Items
+00 6c6561665f30      00 6c6561665f31    01 6c6561665f32      03 6c6561665f33  Leaves with namespace IDs
```

Comment:
```suggestion
00 6c6561665f30      00 6c6561665f31    01 6c6561665f32      03 6c6561665f33  Leaves with namespaces
```

Reply:
Addressed in https://github.com/celestiaorg/nmt/pull/209/commits/5946b91c6735ad609229942334882505f3e3ad79